bin/templatemap_generator.php

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
```

Comment:
@RalfEggert I like that idea; that puts it in the same repo that would consume what it generates.

If you do this, don't forget to add the `bin` entry for it in the `composer.json` (per the [Vendor Binaries documentation](https://getcomposer.org/doc/articles/vendor-binaries.md)). Usage would be like any other installed binary: `./vendor/bin/templatemap_generator.php ...`.

Ping me when you have a PR! :smile: